script/server

```diff
@@ -87,7 +87,7 @@ if [ $? -eq 0 ]; then
 fi
 
 # Check if bundle is complete
-if ! bundle check > /dev/null
+if ! bin/bundle check > /dev/null
```

Comment:
Sorry, this was a relict from my local installation. You may merge with my branch [fix-script-server](https://github.com/spixi/diaspora/tree/fix-script-server) which contains only that change.